socialdistribution/templates/posts/posts_base.html

```diff
@@ -11,10 +11,10 @@
         <div class="post-heading">
             <div class="post-author">
                 <div class="post-author-img">
-                    <img src="/media/{{ post.author.profile_img }}"  alt="">
```

Comment:
Why did we use temp.jpg instead of the author's profile image here?

Reply:
Because remote authors dont have profiles?

Reply:
It is default to temp.jpg in the author model, but are we gonna store remote author in our db?

Reply:
No, we aren't storing remote authors in our db, we just get the response from the api and forward it to the templates.